src/ape/managers/converters.py

```diff
@@ -184,23 +169,39 @@ class ConversionManager:
     plugin_manager: ClassVar[PluginManager] = injected_before_use()  # type: ignore
     networks: ClassVar[NetworkManager] = injected_before_use()  # type: ignore
 
+    def __init__(self) -> None:
+        """
+        Handles the injected properties for ConverterAPI classes
+        """
+        for converter in (
+            AddressAPIConverter,
+            HexAddressConverter,
+            HexConverter,
+            ListTupleConverter,
+            TimestampConverter,
+        ):
+            converter.config = self.config  # type: ignore
+            converter.networks = self.networks  # type: ignore
+            converter.converter = self  # type: ignore
+
```

Comment:
Note: none of the above actually needs any of these values to be set (which is why they were set to `None`), the dependency injection shouldn't cause the class to fail to work, only if the class tries to access that property will it raise.

tl;dr: delete this, it should still function just the same

Reply:
hold on, actually try this instead
```suggestion
    def __init__(self) -> None:
        """
        Handles the injected properties for ConverterAPI classes
        """
        ConverterAPI.config = self.config  # type: ignore
        ConverterAPI.networks = self.networks  # type: ignore
        ConverterAPI.converter = self  # type: ignore
        
```